client/snapshotter_opts_unix.go

```diff
@@ -31,17 +31,33 @@ import (
 )
 
 const (
-	capaRemapIDs     = "remap-ids"
-	capaOnlyRemapIDs = "only-remap-ids"
+	capaRemapIDs      = "remap-ids"
+	capaOnlyRemapIDs  = "only-remap-ids"
+	capaMultiRemapIDs = "multi-remap-ids"
```

Comment:
What is the difference between this 3 modes? only-remap-ids and multi-remap-ids seem to cover all the cases, what is remap-ids doing?

Reply:
I added `multi-remap-ids` instead of replacing `remap-ids` because of backward compat. Some out-of-process snapshotters like `fuse-overlayfs-snapshotter` would break if multiple mappings are configured. So I think we can rely on the different caps to allow the snapshotters to report if multiple mapping entries are supported.

So a few scenarios:
1. Built-in overlayfs is updated to support multiple mappings, so it will report `multi-remap-ids` cap;
2. `fuse-overlayfs-snapshotter` can continue to report `remap-ids`, indicating only single mapping is supported, until it's updated to support multiple entries.

Reply:
Discussed offline to reuse the existing 'remap-ids' cap.